Visual Studio Code (1.74.1, undefined, desktop)
Jupyter Extension Version: 2022.11.1003412109.
Python Extension Version: 2022.20.1.
Workspace folder /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer
info 17:21:41.599: ZMQ install verified.
User belongs to experiment group 'jupyterTestcf'
User belongs to experiment group 'jupyterEnhancedDataViewer'
info 17:21:42.091: LSP Notebooks experiment is enabled
info 17:21:42.718: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 486ms
info 17:21:42.718: Got env vars from Python Ext faster /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python with env var count 70 in 487ms
info 17:21:42.729: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m pip list
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m pip list
info 23:42:35.212: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m pip list
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m pip list
info 23:42:35.251: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 166ms
info 09:12:22.162: Find preferred kernel for /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb with metadata {"colab":{"collapsed_sections":["6710d7b3-fa86-4a5b-8d13-62564d1b2f6f","83563f1b-8033-472c-a0b3-f80ab3bca4d3","89b9c60c-7a43-45f6-b0a7-6d6f7168694a","78fda0a9-19b5-41e3-a643-2174719c86e5","9c787103","8fa24e34","79d40667","oPg7YD-cByNv","X_Of2RuFByN3","PcgPtowxByOB"],"provenance":[]},"kernelspec":{"display_name":"Python 3.10.7 ('venv': venv)","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.10.7"},"vscode":{"interpreter":{"hash":"1744f60b88423f066cba7163e09e742091c4b9dadbc27934e3260623efbc578c"}}} & preferred interpreter /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python
info 09:12:22.167: Preferred kernel .jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher is exact match or top match for non python kernels, (false, [object Object], false, false)
info 09:12:22.169: PreferredConnection: .jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher found for NotebookDocument: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb
info 09:12:27.485: Starting Jupyter Session startUsingPythonInterpreter, .jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher (Python Path: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv, EnvType: Venv, EnvName: 'venv', Version: 3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0]) for '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' (disableUI=false)
info 09:12:27.489: Computing working directory for resource '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 09:12:27.516: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
info 09:12:27.564: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9003 --control=9001 --hb=9000 --Session.signature_scheme="hmac-sha256" --Session.key=b"f544792f-7ea6-4fc3-a2c4-36b589f17101" --shell=9002 --transport="tcp" --iopub=9004 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-2758uRIweN6Vf6tC.json
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9003 --control=9001 --hb=9000 --Session.signature_scheme="hmac-sha256" --Session.key=b"f544792f-7ea6-4fc3-a2c4-36b589f17101" --shell=9002 --transport="tcp" --iopub=9004 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-2758uRIweN6Vf6tC.json
info 09:12:27.565: Process Execution: cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
warn 09:12:27.591: Stderr output when getting ipykernel version & path Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'ipykernel' for /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python
warn 09:12:27.605: StdErr from Kernel Process /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher

error 09:12:27.613: Disposing kernel process due to an error [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
error 09:12:27.619: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher

info 09:12:27.619: Dispose Kernel process 179521.
error 09:12:27.629: Kernel Launcher. launch, Class name = r, completed in 135ms, has a falsy return value, Arg 1: {"id":".jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher","kernelSpec":{"interpreterPath":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python","isRegisteredByVSC":"registeredByNewVersionOfExt","name":"pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f","argv":["python","-m","ipykernel_launcher","-f","{connection_file}"],"language":"python","executable":"python","display_name":"venv","metadata":{"interpreter":{"path":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python"}},"env":{}},"interpreter":{"id":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python","sysPrefix":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","envPath":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","displayPath":{"$mid":1,"fsPath":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","path":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","scheme":"file"},"envName":"venv","displayName":"venv","envType":"Venv","version":{"major":3,"minor":10,"patch":8,"raw":"3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0]"},"path":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python"},"kind":"startUsingPythonInterpreter"}, Arg 2: 60000, Arg 3: "~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb", Arg 4: "~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp", Arg 5: <argument cannot be serialized for logging> [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
error 09:12:27.631: Failed to connect raw kernel session: Error: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
warn 09:12:27.633: Error occurred while trying to start the kernel, options.disableUI=false [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
warn 09:12:27.635: Kernel Error, context = start [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
info 09:12:27.661: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
info 09:12:27.775: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m pip list
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m pip list
info 09:12:27.789: installMissingDependencies /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, ui.disabled=false for resource '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 09:12:27.828: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
info 09:12:28.098: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import pip"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import pip"
info 09:12:28.230: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 736ms
info 09:12:28.233: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 692ms
info 09:12:28.252: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 713ms
info 09:12:28.283: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 646ms
info 09:12:28.289: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 499ms
info 09:12:28.362: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 611ms
info 09:12:28.406: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 344ms
info 09:12:33.317: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import pip"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import pip"
info 09:12:33.410: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:12:33.432: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m pip install -U --force-reinstall ipykernel
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m pip install -U --force-reinstall ipykernel
info 09:12:33.496: Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main

info 09:12:33.496:     return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/lib/python3.10/site-packages/pip/__main__.py", line 9, in <module>
    if sys.path[0] in ("", os.getcwd()):
FileNotFoundError: [Errno 2] No such file or directory

error 09:12:33.514: Failed to install ipykernel     return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/lib/python3.10/site-packages/pip/__main__.py", line 9, in <module>
    if sys.path[0] in ("", os.getcwd()):
FileNotFoundError: [Errno 2] No such file or directory

info 09:12:33.518: Dispose Kernel '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' associated with '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
error 09:12:33.523: Error in execution [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
error 09:12:33.528: Error in execution (get message for cell) [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
info 09:12:33.532: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:12:33.551: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
info 09:12:37.336: Starting Jupyter Session startUsingPythonInterpreter, .jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher (Python Path: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv, EnvType: Venv, EnvName: 'venv', Version: 3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0]) for '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' (disableUI=false)
info 09:12:37.337: Computing working directory for resource '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 09:12:37.340: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 2ms
info 09:12:37.357: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
info 09:12:37.379: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 6ms
info 09:12:37.380: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 5ms
info 09:12:37.410: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9011 --control=9009 --hb=9008 --Session.signature_scheme="hmac-sha256" --Session.key=b"9a1b84b9-d8bd-4a52-ab81-fbfebcb4bc71" --shell=9010 --transport="tcp" --iopub=9012 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-2758MPgNjWzG2CRM.json
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9011 --control=9009 --hb=9008 --Session.signature_scheme="hmac-sha256" --Session.key=b"9a1b84b9-d8bd-4a52-ab81-fbfebcb4bc71" --shell=9010 --transport="tcp" --iopub=9012 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-2758MPgNjWzG2CRM.json
info 09:12:37.410: Process Execution: cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
warn 09:12:37.436: Stderr output when getting ipykernel version & path Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'ipykernel' for /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python
warn 09:12:37.470: StdErr from Kernel Process /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher

error 09:12:37.484: Disposing kernel process due to an error [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
error 09:12:37.485: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher

info 09:12:37.485: Dispose Kernel process 182711.
error 09:12:37.488: Kernel Launcher. launch, Class name = r, completed in 150ms, has a falsy return value, Arg 1: {"id":".jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher","kernelSpec":{"interpreterPath":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python","isRegisteredByVSC":"registeredByNewVersionOfExt","name":"pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f","argv":["python","-m","ipykernel_launcher","-f","{connection_file}"],"language":"python","executable":"python","display_name":"venv","metadata":{"interpreter":{"path":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python"}},"env":{}},"interpreter":{"id":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python","sysPrefix":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","envPath":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","displayPath":{"$mid":1,"fsPath":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","path":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","scheme":"file"},"envName":"venv","displayName":"venv","envType":"Venv","version":{"major":3,"minor":10,"patch":8,"raw":"3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0]"},"path":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python"},"kind":"startUsingPythonInterpreter"}, Arg 2: 60000, Arg 3: "~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb", Arg 4: "~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp", Arg 5: <argument cannot be serialized for logging> [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
error 09:12:37.489: Failed to connect raw kernel session: Error: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
warn 09:12:37.491: Error occurred while trying to start the kernel, options.disableUI=false [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
warn 09:12:37.492: Kernel Error, context = start [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
info 09:12:37.498: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 5ms
info 09:12:37.520: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
info 09:12:37.587: installMissingDependencies /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, ui.disabled=false for resource '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 09:12:37.595: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 8ms
info 09:12:37.613: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
info 09:12:37.675: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:12:37.688: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import pip"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import pip"
info 09:12:39.829: Dispose Kernel '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' associated with '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
error 09:12:39.833: Error in execution [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
error 09:12:39.834: Error in execution (get message for cell) [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
info 09:12:39.836: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 1ms
info 09:12:39.850: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
info 09:12:52.164: Starting Jupyter Session startUsingPythonInterpreter, .jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher (Python Path: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv, EnvType: Venv, EnvName: 'venv', Version: 3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0]) for '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' (disableUI=false)
info 09:12:52.164: Computing working directory for resource '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 09:12:52.169: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:12:52.183: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
info 09:12:52.193: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 1ms
info 09:12:52.194: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 2ms
info 09:12:52.209: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9019 --control=9017 --hb=9016 --Session.signature_scheme="hmac-sha256" --Session.key=b"0a96a9fb-b253-4781-98c3-699016b34e4b" --shell=9018 --transport="tcp" --iopub=9020 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-27585hgHsv6M2OFP.json
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9019 --control=9017 --hb=9016 --Session.signature_scheme="hmac-sha256" --Session.key=b"0a96a9fb-b253-4781-98c3-699016b34e4b" --shell=9018 --transport="tcp" --iopub=9020 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-27585hgHsv6M2OFP.json
info 09:12:52.209: Process Execution: cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
warn 09:12:52.220: Stderr output when getting ipykernel version & path Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'ipykernel' for /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python
warn 09:12:52.238: StdErr from Kernel Process /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher

error 09:12:52.247: Disposing kernel process due to an error [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
error 09:12:52.247: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher

info 09:12:52.247: Dispose Kernel process 182717.
error 09:12:52.251: Kernel Launcher. launch, Class name = r, completed in 85ms, has a falsy return value, Arg 1: {"id":".jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher","kernelSpec":{"interpreterPath":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python","isRegisteredByVSC":"registeredByNewVersionOfExt","name":"pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f","argv":["python","-m","ipykernel_launcher","-f","{connection_file}"],"language":"python","executable":"python","display_name":"venv","metadata":{"interpreter":{"path":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python"}},"env":{}},"interpreter":{"id":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python","sysPrefix":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","envPath":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","displayPath":{"$mid":1,"fsPath":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","path":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv","scheme":"file"},"envName":"venv","displayName":"venv","envType":"Venv","version":{"major":3,"minor":10,"patch":8,"raw":"3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0]"},"path":"/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python"},"kind":"startUsingPythonInterpreter"}, Arg 2: 60000, Arg 3: "~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb", Arg 4: "~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp", Arg 5: <argument cannot be serialized for logging> [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
error 09:12:52.252: Failed to connect raw kernel session: Error: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
warn 09:12:52.253: Error occurred while trying to start the kernel, options.disableUI=false [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
warn 09:12:52.254: Kernel Error, context = start [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
info 09:12:52.259: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 4ms
info 09:12:52.271: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
info 09:12:52.315: installMissingDependencies /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, ui.disabled=false for resource '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 09:12:52.318: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:12:52.332: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
info 09:12:52.362: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 1ms
info 09:12:52.371: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import pip"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import pip"
info 09:12:55.556: Dispose Kernel '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' associated with '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
error 09:12:55.558: Error in execution [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
error 09:12:55.559: Error in execution (get message for cell) [o [Error]: The kernel died. Error: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2162610)
	at ChildProcess.emit (node:events:538:35)
	at ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: f {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: undefined,
      interpreterPath: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'venv',
      metadata: [Object],
      env: {},
      interrupt_mode: undefined
    },
    interpreter: {
      id: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python',
      sysPrefix: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv',
      envPath: [d],
      displayPath: [d],
      envName: 'venv',
      uri: [d],
      displayName: 'venv',
      envType: 'Venv',
      version: [Object]
    },
    id: '.jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python: No module named ipykernel_launcher\n'
}
info 09:12:55.562: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:12:55.575: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel"
info 09:13:30.646: Starting Jupyter Session startUsingPythonInterpreter, .jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher (Python Path: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv, EnvType: Venv, EnvName: 'venv', Version: 3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0]) for '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' (disableUI=false)
info 09:13:30.647: Computing working directory for resource '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 09:13:30.652: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:13:30.669: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
info 09:13:30.686: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:13:30.687: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:13:30.717: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9027 --control=9025 --hb=9024 --Session.signature_scheme="hmac-sha256" --Session.key=b"4e402870-8596-40d5-8536-686d88a979a8" --shell=9026 --transport="tcp" --iopub=9028 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-2758kyq37JngWfUC.json
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9027 --control=9025 --hb=9024 --Session.signature_scheme="hmac-sha256" --Session.key=b"4e402870-8596-40d5-8536-686d88a979a8" --shell=9026 --transport="tcp" --iopub=9028 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-2758kyq37JngWfUC.json
info 09:13:30.718: Process Execution: cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
info 09:13:31.174: ipykernel version & path 6.19.2, ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/lib/python3.10/site-packages/ipykernel/__init__.py for /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python
info 09:13:31.692: Kernel Output: To connect another client to this kernel, use: --existing kernel-v2-2758kyq37JngWfUC.json

info 09:13:31.764: Started Kernel venv (Python 3.10.8) (pid: 192402)
info 09:13:31.768: UpdateWorkingDirectoryAndPath in Kernel
info 09:13:43.147: Restart kernel execution
info 09:13:43.148: Cancel all remaining cells true || Idle || undefined
info 09:13:43.148: Restart requested file:///home/sri/Documents/UTS%20Research/Hugging%20Face%20Spaces/arguementation-analyzer/temp/main.ipynb
info 09:13:43.149: Restarting 3705e2c5-64d0-4aec-aa20-f79d7e4dcf2b
info 09:13:43.153: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:13:43.172: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
info 09:13:43.190: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 3ms
info 09:13:43.191: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 4ms
info 09:13:43.214: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9032 --control=9030 --hb=9029 --Session.signature_scheme="hmac-sha256" --Session.key=b"2ecc9dff-acc7-41ee-bf0b-d743fc98caa5" --shell=9031 --transport="tcp" --iopub=9033 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-2758D9yTsD69EAgj.json
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9032 --control=9030 --hb=9029 --Session.signature_scheme="hmac-sha256" --Session.key=b"2ecc9dff-acc7-41ee-bf0b-d743fc98caa5" --shell=9031 --transport="tcp" --iopub=9033 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-2758D9yTsD69EAgj.json
info 09:13:43.214: Process Execution: cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
info 09:13:43.625: ipykernel version & path 6.19.2, ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/lib/python3.10/site-packages/ipykernel/__init__.py for /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python
warn 09:13:44.096: StdErr from Kernel Process /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/lib/python3.10/site-packages/traitlets/traitlets.py:2544: FutureWarning: Supporting extra quotes around strings is deprecated in traitlets 5.0. You can use 'hmac-sha256' instead of '"hmac-sha256"' if you require traitlets >=5.
  warn(

warn 09:13:44.097: StdErr from Kernel Process /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/lib/python3.10/site-packages/traitlets/traitlets.py:2495: FutureWarning: Supporting extra quotes around Bytes is deprecated in traitlets 5.0. Use '2ecc9dff-acc7-41ee-bf0b-d743fc98caa5' instead of 'b"2ecc9dff-acc7-41ee-bf0b-d743fc98caa5"'.
  warn(

info 09:13:44.105: Kernel Output: To connect another client to this kernel, use: --existing kernel-v2-2758D9yTsD69EAgj.json

info 09:13:44.245: Got new session a74fc2cd-c39c-4abd-8c38-765365b80d34
info 09:13:44.245: Started new restart session
info 09:13:44.246: Dispose Kernel process 192402.
error 09:13:44.277: Error in waiting for cell to complete [Error: Canceled future for execute_request message before replies were done
	at t.KernelShellFutureHandler.dispose (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:32353)
	at /home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:51405
	at Map.forEach (<anonymous>)
	at y._clearKernelState (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:51390)
	at y.dispose (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:44872)
	at /home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2184547
	at t.swallowExceptions (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:7:125449)
	at p.dispose (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2184525)
	at t.RawSession.dispose (/home/sri/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/out/extension.node.js:2:2189519)]
warn 09:13:44.278: Cell completed with errors {
  message: 'Canceled future for execute_request message before replies were done'
}
info 09:13:44.303: UpdateWorkingDirectoryAndPath in Kernel
warn 09:13:55.799: StdErr from Kernel Process 2022-12-18 09:13:55.797547: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.

warn 09:13:56.960: StdErr from Kernel Process 2022-12-18 09:13:56.552171: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2022-12-18 09:13:56.552209: I tensorflow/compiler/xla/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.

warn 09:13:58.675: StdErr from Kernel Process 2022-12-18 09:13:58.670626: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory

warn 09:13:58.675: StdErr from Kernel Process 2022-12-18 09:13:58.670916: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory
2022-12-18 09:13:58.670935: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.

info 09:14:28.179: Cancel all remaining cells true || Error || undefined
info 09:46:21.817: Disposing kernel .jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher for notebook /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb due to selection of another kernel or closing of the notebook
info 09:46:21.818: Dispose Kernel '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' associated with '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 09:46:21.821: Dispose Kernel process 196287.
info 09:52:24.016: Ending debug session 8e8ca801-52a8-4215-b8eb-b0f3f77e12fb
info 09:52:24.016: Ending debug session 8e8ca801-52a8-4215-b8eb-b0f3f77e12fb
info 09:53:14.369: Ending debug session 89f0eeb6-1f1f-48ef-bdb9-b6a9b63ea690
info 09:53:14.369: Ending debug session 89f0eeb6-1f1f-48ef-bdb9-b6a9b63ea690
info 09:53:52.446: Ending debug session 51f28734-d2d2-4de2-90ca-e849d5e0e3f5
info 09:53:52.446: Ending debug session 51f28734-d2d2-4de2-90ca-e849d5e0e3f5
info 09:55:21.108: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m pip list
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m pip list
info 09:55:21.359: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 273ms
info 09:55:30.331: Starting Jupyter Session startUsingPythonInterpreter, .jvsc74a57bd0e4873d6a3d5509a84168c34f7a06690d6b9af20b68e3cf312a0f4ae94fe8b47f./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python./home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/python.-m#ipykernel_launcher (Python Path: /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv, EnvType: Venv, EnvName: 'venv', Version: 3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0]) for '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' (disableUI=false)
info 09:55:30.332: Computing working directory for resource '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 09:55:30.351: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
info 09:55:30.398: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9027 --control=9025 --hb=9024 --Session.signature_scheme="hmac-sha256" --Session.key=b"cf454dda-80eb-47b4-9b65-a0f9c8982716" --shell=9026 --transport="tcp" --iopub=9028 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-27588rbyWOzt3crZ.json
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9027 --control=9025 --hb=9024 --Session.signature_scheme="hmac-sha256" --Session.key=b"cf454dda-80eb-47b4-9b65-a0f9c8982716" --shell=9026 --transport="tcp" --iopub=9028 --f=/home/sri/.local/share/jupyter/runtime/kernel-v2-27588rbyWOzt3crZ.json
info 09:55:30.398: Process Execution: cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
cwd: ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp
info 09:55:30.563: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 228ms
info 09:55:30.581: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 214ms
info 09:55:30.590: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 219ms
info 09:55:30.810: ipykernel version & path 6.19.2, ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/lib/python3.10/site-packages/ipykernel/__init__.py for /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python
warn 09:55:31.521: StdErr from Kernel Process /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/lib/python3.10/site-packages/traitlets/traitlets.py:2544: FutureWarning: Supporting extra quotes around strings is deprecated in traitlets 5.0. You can use 'hmac-sha256' instead of '"hmac-sha256"' if you require traitlets >=5.
  warn(

warn 09:55:31.521: StdErr from Kernel Process /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/lib/python3.10/site-packages/traitlets/traitlets.py:2495: FutureWarning: Supporting extra quotes around Bytes is deprecated in traitlets 5.0. Use 'cf454dda-80eb-47b4-9b65-a0f9c8982716' instead of 'b"cf454dda-80eb-47b4-9b65-a0f9c8982716"'.
  warn(

info 09:55:31.534: Kernel Output: To connect another client to this kernel, use: --existing kernel-v2-27588rbyWOzt3crZ.json

info 09:55:31.666: Started Kernel venv (Python 3.10.8) (pid: 239139)
info 09:55:31.669: UpdateWorkingDirectoryAndPath in Kernel
info 09:55:31.709: Got env vars with python /home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python, with env var count 70 and custom env var count 66 in 2ms
info 09:55:31.724: Process Execution: > ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python ~/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/pythonFiles/printJupyterDataDir.py
> ~/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/venv/bin/python ~/.vscode/extensions/ms-toolsai.jupyter-2022.11.1003412109/pythonFiles/printJupyterDataDir.py
warn 09:55:45.165: StdErr from Kernel Process 2022-12-18 09:55:44.797751: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.

warn 09:55:45.166: StdErr from Kernel Process 2022-12-18 09:55:45.072883: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2022-12-18 09:55:45.072922: I tensorflow/compiler/xla/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.

warn 09:55:46.584: StdErr from Kernel Process 2022-12-18 09:55:46.581145: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory

warn 09:55:46.584: StdErr from Kernel Process 2022-12-18 09:55:46.581281: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory
2022-12-18 09:55:46.581302: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.

info 10:01:54.741: Ending debug session ecf303b1-a7f0-4cf0-97ea-64061f38a245
info 10:01:54.741: Ending debug session ecf303b1-a7f0-4cf0-97ea-64061f38a245
info 10:02:05.455: Ending debug session dd65473e-955c-49a8-9798-ff0d4d64cd65
info 10:02:05.455: Ending debug session dd65473e-955c-49a8-9798-ff0d4d64cd65
info 10:02:28.278: Ending debug session 925b544a-362e-4e83-a148-4b5e9ef2fc92
info 10:02:28.278: Ending debug session 925b544a-362e-4e83-a148-4b5e9ef2fc92
info 10:05:49.134: Ending debug session d552f44f-b7f7-4d65-a3ae-fbd366a07e55
info 10:05:49.135: Ending debug session d552f44f-b7f7-4d65-a3ae-fbd366a07e55
info 11:06:15.521: Shutting down notebooks for 1b683db9-0b08-4a10-ad8c-e0cb92874812
info 11:06:15.524: Disposing HostJupyterExecution 4dd94de7-f952-42c5-b1b7-f99fe0646660
info 11:06:15.524: Disposing super HostJupyterExecution 4dd94de7-f952-42c5-b1b7-f99fe0646660
info 11:06:15.525: Dispose Kernel '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' associated with '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 11:06:15.525: Dispose Kernel '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' associated with '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 11:06:15.525: Dispose Kernel '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' associated with '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 11:06:15.526: Dispose Kernel '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' associated with '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 11:06:15.526: Dispose Kernel '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb' associated with '/home/sri/Documents/UTS Research/Hugging Face Spaces/arguementation-analyzer/temp/main.ipynb'
info 11:06:15.531: Dispose Kernel process 239139.
info 11:06:15.563: Disposing HostJupyterExecution 4dd94de7-f952-42c5-b1b7-f99fe0646660
info 11:06:15.563: Finished disposing HostJupyterExecution  4dd94de7-f952-42c5-b1b7-f99fe0646660
info 11:06:15.582: Cleaning up server cache 4dd94de7-f952-42c5-b1b7-f99fe0646660
info 11:06:15.592: Finished disposing HostJupyterExecution  4dd94de7-f952-42c5-b1b7-f99fe0646660
